src/Runner.Worker/Handlers/ScriptHandler.cs

```diff
@@ -156,6 +144,16 @@ public async Task RunAsync(ActionRunStage stage)
             var githubContext = ExecutionContext.ExpressionValues["github"] as GitHubContext;
             ArgUtil.NotNull(githubContext, nameof(githubContext));
 
+            // Add Telemetry to JobContext to send with JobCompleteMessage
+            if (stage == ActionRunStage.Main)
+            {
+                var telemetry = new ActionsStepTelemetry {
```

Comment:
Consider adding a way to determine what shell customers are using to execute scripts.

Reply:
Yeah, not the input itself, but just the "entrypoint".